### PR TITLE
Mohammad-Darwesh homework js2 week1

### PR DESCRIPTION
**Pull request by mohammadnaefaldarwesh**

### CODE REVIEW

File: Week1/Mohammad-Darwesh-JS2-week1/app.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      'https://images-na.ssl-images-amazon.com/images/I/5116zAISjzL._SX329_BO1,204,203,200_.jpg',
+    church_of_marvels:
+      'https://i.pinimg.com/originals/b8/19/0d/b8190d0fb7bcb1ced1bc4d69c3637c22.jpg',
+    after_the_rain:
+      'https://vicsmediaroom.files.wordpress.com/2016/04/after-the-rain-book-cover.jpg?w=308',
+  };
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+  const header = document.createElement('h1');
+  const addHeader = document.body.getElementsByTagName('div')[0];
```
Comment (rohni) [inline Week1/Mohammad-Darwesh-JS2-week1/app.js]: Just to remember if another div is added to the page somewhere, this code will break.  For this sort of thing I would have given the div an `id`.

File: Week1/Mohammad-Darwesh-JS2-week1/app.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  addHeader.appendChild(header);
+  header.innerHTML = 'My favorite books';
+  const list = document.createElement('ul');
+  div.appendChild(list);
+  list.setAttribute('class', 'books_list');
+  function makingList() {
```
Comment (rohni) [inline Week1/Mohammad-Darwesh-JS2-week1/app.js]: Nice that you broke this into a function

File: Week1/Mohammad-Darwesh-JS2-week1/app.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    for (let item of bookTitles) {
```
Comment (rohni) [inline Week1/Mohammad-Darwesh-JS2-week1/app.js]: I would have taken the contents of the loop and created another function where you create one complete list item, and then just call that function in the loop and pass in the book data.

File: Week1/Mohammad-Darwesh-JS2-week1/app.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      const booksList = document.createElement('li');
+      list.appendChild(booksList);
+      booksList.setAttribute('id', item);
```
Comment (rohni) [inline Week1/Mohammad-Darwesh-JS2-week1/app.js]: You have a function later that inserts the cover (as should happen from the spec) why are you adding it here as well?

Comment (mohammadnaefaldarwesh) [reply Week1/Mohammad-Darwesh-JS2-week1/app.js]: Hello Rohni ,
Thank you for your  feedback and sure i will consider your useful advises.
about insert the cover ......my idea from tow function that one for HTML structures(elements,tags,attributes) and the second one  for insert stuff in the elements.

**Pull request closed without merging**